src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const validActiveTab = tabs.find(tab => tab.id === activeTabId) ?? tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn({ 'is-active': validActiveTab.id === tab.id })}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() =>
+                  validActiveTab.id !== tab.id ? onTabSelected(tab.id) : null
```

Comment:
Create a variable for boolean condition to avoid duplication. Also, consider using && or `if` statement instead of the ternary operator